test/bundling/webpack/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+const config = {
+  entry: './src/index.ts',
+  output: {
+    path: path.resolve(__dirname, 'dist')
+  },
+  experiments: { topLevelAwait: true },
+  target: 'node',
```

Comment:
This line in combination with lines 20 & 21 are they key to get the bundle to work.